tests/OmniSharp.Roslyn.CSharp.Tests/HighlightFacts.cs

```diff
@@ -7,178 +7,146 @@
 using Xunit;
```

Comment:
Changes to this file should be reverted. The OmniSharp v1 Highlight service is not intended to drive SemanticHighlight. It may still be used by some Editor extensions.